country-dashboard/src/index.css

```diff
@@ -1,13 +1,65 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+*,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p
+ {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+html {
+  max-width: 1280px;
+  margin: auto;
 }
+body
+{
+ max-width:1100px;
+ margin: auto;
+}
+
+.App {
+  display: flex;
+  flex-direction: column;
+  gap: 50px;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding-left: 50px;
+}
+.header-Search
+{
+  display: flex;
+  gap: 20px;
+
+}
+.main-Content {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 20px;
+}
+.card--ui {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  gap: 10px;
+  border: 1px solid black;
+  padding: 10px;
+}
+.card-text {
```

Comment:
I did say that you don't need to exactly replicate the UI in the figma, but this is quite far off from the design in the figma. You could have had rounded corners on the card, more padding in the card, a lighter border, use a similar font and bolding etc. -3 

<img width="234" alt="image" src="https://github.com/rumoravenue/rumor-react-training-a2/assets/62576642/4779c6d3-f529-4984-93be-9cf589e3e6e7">